### PR TITLE
Some updates :-)

### PR DESCRIPTION
**Pull request by Sanix-Darker**

### CODE REVIEW

File: atbswp/utils.py [hunk 1]
```diff
@@ -17,10 +17,6 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import os
-import configparser
```
Comment (RMPR) [inline atbswp/utils.py]: I don't really understand why the changes on this file.

File: atbswp/control.py [hunk 2]
```diff
@@ -147,49 +169,50 @@ def on_press(self, key):
 
         try:
             self.capture.append(f"pyautogui.keyDown({repr(key.char)})")
+        except AttributeError as es:
+            print("AttributeError : ", es)
```
Comment (RMPR) [inline atbswp/control.py]: printing is not really required here because the fact that an attribute error occurs means a special character has been closed, hence the additional tests

File: atbswp/control.py [hunk 3]
```diff
@@ -100,6 +100,28 @@ def __init__(self):
 
         self.capture = [self.header]
 
+
```
Comment (RMPR) [inline atbswp/control.py]: I really like this refactoring, maybe give more meaningful names to the functions ? Idk something like write_keyboard_actions/write_mouse_actions.

File: atbswp/control.py [hunk 4]
```diff
@@ -100,6 +100,28 @@ def __init__(self):
...
+    def clk_mouse(self, mode, param):
```
Comment (RMPR) [inline atbswp/control.py]: I really like this refactoring, maybe give more meaningful names to the functions ? Idk something like write_keyboard_actions/write_mouse_actions.

File: atbswp/control.py [hunk 5]
```diff
@@ -100,6 +100,28 @@ def __init__(self):
...
+    def clk_kboard(self, mode, param):
+        """
+        Params:
+            mode : Defines the mode of the action ( down, up )
+            param : Dfines the parameter for pyautogui
+
+            The .title() will make down to Down
+        """
+        self.capture.append(f"pyautogui.key"+mode.title()+"('"+param+"')")
+
```
Comment (RMPR) [inline atbswp/control.py]: I really like this refactoring, maybe give more meaningful names to the functions ? Idk something like write_keyboard_actions/write_mouse_actions.

File: requirements-dev.txt [hunk 6]
```diff
@@ -2,7 +2,7 @@ altgraph==0.16.1
 attrs==19.3.0
 CacheControl==0.12.5
 cachy==0.3.0
-certifi==2019.11.28
+# certifi==2019.11.28
```
Comment (RMPR) [inline requirements-dev.txt]: Why ?

Comment (wilcoln) [reply requirements-dev.txt]: Dit-il d'un air ébailli
